scripts/config.ts

```diff
@@ -88,7 +88,7 @@ const config = {
   // ----------------------------------
   // Server Configuration
   // ----------------------------------
-  server_host: 'localhost',
+  server_host: '0.0.0.0',
```

Comment:
Please revert this change before merging

Reply:
Done.